### PR TITLE
Feat/credit modal

### PR DESCRIPTION
**Pull request by KyuubiNoKitsune**

### CODE REVIEW

File: app/javascript/components/payment-error-button.vue [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
         </div>
-        <div v-if="showCartOnDebtModal">
-          <h1> Productos </h1>
+        <div class="invoice__apology-modal-text-second" v-if="showCartOnDebtModal">
+          <h5> Productos </h5>
           <cartProducts/>
         </div>
         <div v-if="showDebtFinalMessage">
-          <h1> Listo! </h1>
+          <h5> Listo! </h5>
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 12 spaces but found 10 spaces  vue/html-indent

File: app/javascript/components/payment-error-button.vue [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-        <div v-if="showApologyText">
-          <h1> Error en el pago </h1>
+        <div class="invoice__apology-modal-text-first" v-if="showApologyText">
+          <h5> Error en el pago </h5>
           Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
         </div>
-        <div v-if="showCartOnDebtModal">
-          <h1> Productos </h1>
+        <div class="invoice__apology-modal-text-second" v-if="showCartOnDebtModal">
+          <h5> Productos </h5>
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 12 spaces but found 10 spaces  vue/html-indent

File: app/javascript/components/payment-error-button.vue [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
       <div class="invoice__apology-modal-text">
-        <div v-if="showApologyText">
-          <h1> Error en el pago </h1>
+        <div class="invoice__apology-modal-text-first" v-if="showApologyText">
+          <h5> Error en el pago </h5>
           Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
         </div>
-        <div v-if="showCartOnDebtModal">
-          <h1> Productos </h1>
+        <div class="invoice__apology-modal-text-second" v-if="showCartOnDebtModal">
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 10 spaces but found 8 spaces   vue/html-indent<br>Attribute "if" should be on a new line                 vue/max-attributes-per-line

File: app/javascript/components/payment-error-button.vue [hunk 4]
```diff
@@ -1,17 +1,18 @@
 <template>
   <div class="payment-error-button">
     <modal name="apology-modal">
+      <div class="invoice__apology-modal-flexbox">
       <div class="invoice__apology-modal-text">
-        <div v-if="showApologyText">
-          <h1> Error en el pago </h1>
+        <div class="invoice__apology-modal-text-first" v-if="showApologyText">
+          <h5> Error en el pago </h5>
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 12 spaces but found 10 spaces  vue/html-indent

File: app/javascript/components/payment-error-button.vue [hunk 5]
```diff
@@ -1,17 +1,18 @@
 <template>
   <div class="payment-error-button">
     <modal name="apology-modal">
+      <div class="invoice__apology-modal-flexbox">
       <div class="invoice__apology-modal-text">
-        <div v-if="showApologyText">
-          <h1> Error en el pago </h1>
+        <div class="invoice__apology-modal-text-first" v-if="showApologyText">
```
Comment (monkeyci) [inline app/javascript/components/payment-error-button.vue]: Expected indentation of 10 spaces but found 8 spaces   vue/html-indent<br>Attribute "if" should be on a new line                 vue/max-attributes-per-line

File: app/assets/stylesheets/refreshments/_invoice.scss [hunk 6]
```diff
@@ -22,23 +22,40 @@ $copy-icon-color: #fff;
...
+    padding-right: 10px;
+    padding-top: 10px;
+
+  }
+
+  &__apology-modal-text-first {
+    padding: 30px;
+  }
+
+  &__apology-modal-text-first {
```
Comment (monkeyci) [inline app/assets/stylesheets/refreshments/_invoice.scss]: Unexpected duplicate selector ".invoice__apology-modal-text-first", first used at line 46   no-duplicate-selectors

File: app/assets/stylesheets/refreshments/_debt-product-list.scss [hunk 7]
```diff
@@ -1,5 +1,8 @@
 .debt-product-list {
   display: table;
+  overflow: scroll;
+  height: 150px;
+  display: block;
```
Comment (monkeyci) [inline app/assets/stylesheets/refreshments/_debt-product-list.scss]: Unexpected duplicate "display"   declaration-block-no-duplicate-properties

**Comment by blackjid** [comment]
Hay varias cosas del mono que arreglaria.

**Pull request merged**